packages/date-picker/src/FilterDateRangePicker/FilterDateRangePicker.tsx

```diff
@@ -142,17 +158,101 @@ export const FilterDateRangePicker = ({
     }
   }, [isOpen, rangeStart, rangeEnd])
 
+  const [inbuiltStartDateStatus, setInbuiltStartDateStatus] = useState<
+    ValidationResponse["status"] | undefined
+  >()
+  const [
+    inbuiltStartDateValidationMessage,
+    setInbuiltStartDateValidationMessage,
+  ] = useState<string | undefined>()
+  const [inbuiltEndDateStatus, setInbuiltEndDateStatus] = useState<
+    ValidationResponse["status"] | undefined
+  >()
+  const [inbuiltEndDateValidationMessage, setInbuiltEndDateValidationMessage] =
+    useState<string | undefined>()
```

Comment:
...I think we need to kill this message/status pattern. If the crazy types Nat had to do to make hers work wasn't a sign already, this is just so verbose lol

Reply:
And I think consumers would agree, most of their app errors should be in the shape of
```
{ status, message }
```

Reply:
Yes I completely agree. Definitely keen to get rid of it if we could.
I could probably change these state objects though anyway, at least for inbuilt. It did come to me over the weekend, so I was planning to refactor to do it :)